CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/gcp_api_util.py

```diff
@@ -88,10 +93,91 @@ def build_sql_admin_service(credentials=None):
         credentials or default_credentials
     )
 
+    service = discovery.build(
+        service,
+        version,
+        http=authorized_http
+    )
+
+    return service
+
+# Compute
+def build_compute_service(credentials=None):
+    """Authorizes and sets custom CloudSQL Replicator user agent.
```

Comment:
"Authorizes and sets custom CloudSQL Replicator user agent" as a comment does not match the function name build_compute_service

Reply:
Fixed.